astropy/table/tests/test_table.py

```diff
@@ -496,6 +529,19 @@ def test_add_columns5(self, table_types):
         t.add_columns([self.c, self.d], indexes=[2, 2])
         assert t.colnames == ['a', 'b', 'c', 'd']
 
+    def test_add_columns6(self, table_types):
```

Comment:
Here you're stuck with a previous naming scheme, but I'd still recommend adding a line what the test is for:
```
"""Check we can override column names."""
```